lib/s3routes/routesUtils.js

```diff
@@ -242,54 +247,97 @@ function okContentHeadersResponse(overrideParams, resHeaders,
     return response;
 }
 
-function retrieveData(locations, dataRetrievalFn,
-    response, logger, errorHandlerFn) {
+function retrieveDataAzure(locations, dataRetrievalFn, response, logger) {
```

Comment:
Rename `dataRetrievalFn` to `retrieveDataFn` to match the retrieveData function?